explorer/interpreter/type_checker.cpp

```diff
@@ -4588,6 +4588,17 @@ auto TypeChecker::DeclareClassDeclaration(Nonnull<ClassDeclaration*> class_decl,
       ScopeInfo::ForClassScope(scope_info, &class_scope, std::move(bindings));
   for (Nonnull<Declaration*> m : class_decl->members()) {
     CARBON_RETURN_IF_ERROR(DeclareDeclaration(m, class_scope_info));
+
+    if (const auto* fun = dyn_cast<FunctionDeclaration>(m)) {
+      if (fun->virt_override() == VirtualOverride::Impl) {
+        const VTable& vtable = (*base_class)->vtable();
+        const auto* vtable_fun = vtable[fun->name()].first;
```

Comment:
You'd need to use `.find` here (https://llvm.org/doxygen/classllvm_1_1StringMap.html#a919e3939ef1a80775fef49a43bfff69b), as the `vtable` is `const`